packages/block-editor/src/components/block-list/style.scss

```diff
@@ -1009,3 +600,30 @@
 		margin-bottom: auto;
 	}
 }
+
+.components-popover.block-editor-block-list__block-popover {
+	z-index: z-index(".block-editor-block-list__block-popover");
+
+	.components-popover__content {
+		margin-left: 0 !important;
+		min-width: auto;
+		background: none;
+		border: none;
+		box-shadow: none;
+		overflow-y: visible;
+
+		.block-editor-block-contextual-toolbar,
+		.block-editor-block-list__breadcrumb {
+			margin-bottom: $block-padding - $border-width;
+			margin-left: - $block-padding;
+		}
+
+		.block-editor-block-contextual-toolbar[data-align="full"] {
+			margin-left: 0;
```

Comment:
@ellatrix do you know why we need this? With G2 it's causing a bug (can't click the movers) but I'm not sure if it's needed for something else.